examples/atable/default.story.vue

```diff
@@ -2,6 +2,8 @@
 	<Story title="default">
 		<Variant title="default">
 			<ATable v-model="default_table.rows" :columns="default_table.columns" :config="default_table.config" />
+			<ATable v-model="default_table_2.rows" :columns="default_table_2.columns" :config="default_table_2.config" />
+			<ATable v-model="default_table_3.rows" :columns="default_table_3.columns" :config="default_table_3.config" />
```

Comment:
I think these should go into their own variant; the `default` story variannt should just show a simple table.